### PR TITLE
#2303 Impossible to surf on 3D printing SE ZIM file

### LINKED ISSUE #2303: Impossible to surf on 3D printing SE ZIM file
Steps:
* Download 3D printing ZIM file
* Load it
* Welcome page displyas properly
* clicking on the first link

I get:
* a blank page

I want:
* The proper page

### PR DESCRIPTION
**Pull request by macgills**
Fixes #2303 

- opening a zim file unregisters before loading

### CODE REVIEW

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/ServiceWorkerUninitialiser.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+              }
+              else {
+                ServiceWorkerUninitialiser.onUninitialised();
+              }
+            });
+          }
+        do_unregister();
+        </script>
+    </head>
+    <h1>---</h1>
```
Comment (macgills) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/ServiceWorkerUninitialiser.kt:52]: @rgaudin can you suggest some html to have some type of loading spinner? Or we could just have an entirely blank page

Comment (rgaudin) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/ServiceWorkerUninitialiser.kt:52]: We have https://github.com/openzim/warc2zim/blob/master/src/warc2zim/templates/index.html for WARC2zim, that's probably a quick and easy way to have a visual feedback for now that can be improved later.

Do you need/want me to test your PR?

Comment (macgills) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/ServiceWorkerUninitialiser.kt:52]: From my limited testing the core concept is solid, it wouldn't hurt to test it but I wouldn't make it a priority

**Comment by macgills** [comment]
@abdulwd finally have this fixed, if there are any obstacles to reviewing please let me know

**Review by abdulwd — APPROVED** [review]

**Pull request merged**